steps/configcat-feature-flag-sync/0.0.1/step.yml

```diff
@@ -0,0 +1,76 @@
+  API credentials](https://app.configcat.com/my-account/public-api-credentials/) \n1.
+  Store your credentials in the following Bitrise secret environment variables: `CONFIGCAT_API_USER`,
+  `CONFIGCAT_API_PASS`.\n1. Add your [ConfigCat config ID](https://configcat.com/docs/advanced/code-references/overview#config-id)
+  to the `ConfigCat Config ID` input variable of this step (`configcat_config_id`
+  in bitrise.yml). The config ID tells the step which feature flags it should search
+  for in your source code.\n\nFor more information about code reference scanning,
+  see the [documentation](https://configcat.com/docs/advanced/code-references/overview/).\n\n###
+  Code snippet on the ConfigCat Dashboard\n![Code references](https://configcat.com/docs/assets/cli/scan/scan_report.png)"
+website: https://configcat.com/
+source_code_url: https://github.com/configcat/bitrise-step-configcat-feature-flag-sync/
+support_url: https://configcat.com/support/
+published_at: 2022-05-03T11:09:46.334437+02:00
+source:
+  git: https://github.com/configcat/bitrise-step-configcat-feature-flag-sync.git
+  commit: b1e9ffa6ed6e38499e899fc9088c2c7f13614821
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+inputs:
+- configcat_api_host: api.configcat.com
+  opts:
+    is_required: true
+    summary: Full domain name without protocol e.g. 'api.configcat.com'
+    title: ConfigCat Public Management API host
+- configcat_api_user: $CONFIGCAT_API_USER
+  opts:
+    is_required: true
+    is_sensitive: true
+    summary: Create your ConfigCat Public Management API credentials [here](https://app.configcat.com/my-account/public-api-credentials/)
+    title: ConfigCat Public Management API basic authentication username
+- configcat_api_pass: $CONFIGCAT_API_PASS
+  opts:
+    is_required: true
+    is_sensitive: true
+    title: ConfigCat Public Management API basic authentication password
+- configcat_config_id: null
+  opts:
+    is_required: true
+    summary: The [config ID](https://configcat.com/docs/advanced/code-references/overview#config-id)
+      tells the step which feature flags it should search for in your source code.
+    title: ConfigCat Config ID
+- line_count: 4
+  opts:
+    summary: The ConfigCat Dashboard will display a few lines of your source code
+      around the feature flag reference. You can set here how many lines of code you
+      want to include before and after the reference line.
+    title: 'Code snippet line count before and after the reference line (min: 1, max:
+      10)'
+- opts:
+    summary: e.g. 'subfolder/to/scan'
+    title: Sub-folder to scan, relative to the repository root folder
+  sub_folder: null
+- opts:
+    summary: '`true` or `false`'
+    title: Turns on detailed logging
+  verbose: false
```

Comment:
Step Input Value Options could be used here to ease this input configuration:

```
      value_options:
        - "false"
        - "true"
```

Please surround boolean values with double quotes (both for the value options and default value), to enforce string types.